### ISSUE TITLE
Orthographic Camera & Point Cloud - Point cloud not updating

### ISSUE THREAD

**Issue by Pourfex** [iTowns/itowns#2232]
Orthographic camera usage on itowns makes any point cloud added in the scene not updating.

## Your Environment
* Version used: itowns 2.41.1.9
* Browser Name and version: chrome, firefox
* Operating System and version (desktop or mobile): desktop
* Link to your project:
Exemple here : 
https://65674e4ca6a4df257c21c636--demo-hackathon.netlify.app/examples/#misc_orthographic_camera
Code here : 
https://github.com/bloc-in-bloc/itowns/blob/master/examples/misc_orthographic_camera.html

## Context
Point cloud layer is not updating if we use orthographic camera (perspective camera work). Don't know if anyone has this use case before.
We think it's because of the orthographic camera position which never moves but only "zoom" so point cloud layer has the same position of camera whatever the orthographic camera zoom has.

## Steps to Reproduce (for bugs)
Check exemples, zoom on point cloud with orthographic camera view

## Expected Behavior
Point cloud layer should update based on zoom.

## Actual Behavior
Point cloud layer does not update based on zoom.

## Possible Cause/Fix/Solution
We think it's because of the orthographic camera position which never moves but only "zoom" so point cloud layer has the same position of camera whatever the orthographic camera zoom has.

**Comment by Desplandis** [comment]
I'll investigate this issue.

**Comment by Desplandis** [comment]
Hi and thanks for your detailed bug report, I managed to easily reproduce this issue on the `master` branch!

Since we are using an orthographic camera, the position of the camera never changes while zooming, only the `zoom` property changes. This property factor the top, bottom, right and left planes of the frustrum (but only updates the projection matrix, not the corresponding properties).

Since the point cloud implementation was never used on an orthographic camera context, we will need to:
- have another "sse" computation method which uses the camera projection matrix.
- change the points shader which may be incorrect for the attenuated point calculation.

I have started a branch which "fixes" both issue and I'll soon open a PR, I will tag you for testing! =)
